### ISSUE TITLE
Manual Grader Width of Current Student Menu

### ISSUE THREAD

**Issue by somiaj** [openwebwork/webwork2#1805]
In the manual grader, the width of the current student drop down menu changes as you cycle through students (due to varying lengths of student names). Would it be possible to compute the largest possible width and set the width to that? Thus when cycling through students the forward arrow won't keep moving location.

**Comment by drgrice1** [comment]
I have noticed this at times as well, although I don't find it to serious.  If I want to rapidly cycle through the students, I go to the end of the list, and go back.  The left button never moves.

It may be possible to allay the issue with css styling and setting a minimum width.  However, your suggestion will not work.  You might think that it is as simple as computing a maximum number of characters in names, but it really is much more than that.  This is not a fixed width font, and so that is not reliable.  In addition I have seen attempts at determining a maximum width for words in a particular font, and it always ends up being an unreliable hack.

**Comment by Alex-Jordan** [comment]
Is there a technique where the students names are all there, all but one occupy zero height, all but one are not visible (even to AT users), and yet all widths are accounted for in the div's width? Writing it down, it sounds like along shot.

**Comment by somiaj** [comment]
@drgrice1 Agreed this is a minor issue, and I to have been cycling backwards. I was just seeing if it was possible to fix in a nice way. If not, understandable to leave alone. Another thought would be could we use the width of the drop down menu, since it already incorporates the largest possible name?
